### ISSUE TITLE
Update README.md, instructions to avoid Python installation should be more visible

### ISSUE THREAD

**Issue by ahojukka5** [aws/aws-elastic-beanstalk-cli-setup#100]
In FAQ: "4.1. Can I skip the Python installation?" definitely should be shown in the very first lines in README before people following the instructions and start to install Python.

**Comment by juanitogan** [comment]
I agree... and it should also be corrected.  You need Python, pip, and virtualenv installed... apparently.  Ugh, messy.  And, if I don't go this route, it tries to install Python 3.7 even though I have 3.8 installed.

**Comment by jstrese** [comment]
I saw this after fighting with step 2 for a bit. I totally agree.

In the end all I had to do was the following since I'm on 3.9 -

```
pip3 install virtualenv
python3 aws-elastic-beanstalk-cli-setup/scripts/ebcli_installer.py
```
